plottable.css

```diff
@@ -164,6 +164,14 @@ svg.plottable {
   stroke-width: 1px;
 }
 
+.plottable .drag-line-layer.enabled.vertical line.drag-edge {
+  cursor: ew-resize;
+}
+
+.plottable .drag-line-layer.enabled.horizontal line.drag-edge {
+  cursor: ns-resize;
+}
```

Comment:
When I think of moving though I personally expect the hand cursor.  We can we `DragBoxLayer` as an example